docs/source/gbd2017_models/causes/neonatal_disorders/index.rst

```diff
@@ -0,0 +1,265 @@
+.. _2017_cause_neonatal_disorders:
+
+==================
+Neonatal Disorders
+==================
+
+.. todo::
+
+  Add a brief introductory paragraph for this document.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Disease Overview
+----------------
+
+.. todo::
+
+   Add a general clinical overview of the cause.
+
+
+GBD 2017 Modeling Strategy
+--------------------------
+
+.. todo::
+
+  Add an overview of the GBD modeling section, citing
+  [GBD-2017-YLD-Appendix-Neonatal-Disorders]_ and
+  [GBD-2017-CoD-Appendix-Neonatal-Disorders]_.
+
+
+Cause Hierarchy
++++++++++++++++
+
+Restrictions
+++++++++++++
+
+The following table describes any restrictions in GBD 2017 on the effects of
+this cause (such as being only fatal or only nonfatal), as well as restrictions
+on the ages and sexes to which the cause applies.
+
+.. list-table:: GBD 2017 Cause Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     -
+   * - YLL only
+     - False
+     -
+   * - YLD only
+     - False
+     -
+   * - YLL age group start
+     - Early Neonatal
+     - [0, 7 days), age_group_id=2
+   * - YLL age group end
+     - Post Neonatal
+     - [28 days, 1 year), age_group_id=4
+   * - YLD age group start
+     - Early Neonatal
+     - [0, 7 days), age_group_id=2
+   * - YLD age group end
+     - 95 plus
+     - [95, 125 years), age_group_id=235
+
+
+Vivarium Modeling Strategy
+--------------------------
+
+This model is designed to estimate YLLs due to neonatal disorders that could be
+averted by an intervention that affects :ref:`Low Birth Weight and Short
+Gestation (LBWSG) <2017_risk_lbwsg>`. The model groups together all of the
+neonatal sub-causes, and is fatal-only (no disability). The rationale for this
+design is as follows:
+
+- All of the neonatal sub-causes (except sepsis -- see `Assumptions and
+  Limitations`_ below) have the same 2-state cause model structure: "with
+  condition" and "free of condition", with no between-state transitions.
+
+- All of the causes affected by the LBWSG risk factor have the same risk ratios
```

Comment:
This is a small note and I am lazy about my RR terminology, but maybe we should write "relative risk" here instead of "risk ratio" to make sure that we are being consistent with GBD language.

Reply:
Ok, seems like a good idea to keep it consistent